Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -323,6 +323,10 @@ private void OnElectrifiedInteractUsing(EntityUid uid, ElectrifiedComponent elec
         var electrocutionNode = Comp<NodeContainerComponent>(electrocutionEntity).GetNode<ElectrocutionNode>("electrocution");
         var electrocutionComponent = Comp<ElectrocutionComponent>(electrocutionEntity);
 
+        // This shows up in the power monitor.
+        // Yes. Yes exactly.
+        MetaData(electrocutionEntity).EntityName = MetaData(uid).EntityName;
```

Comment:
Should this use Identity name?

Reply:
honestly it shouldnt use names at all, how the hell does the power monitor know the name or visible name of a mob that gets shocked

Reply:
It's not realistic, but it IS funny / fun. Isn't that good enough?

Reply:
That's the joke

Reply:
Showing the person's name is powerful because you can instantly decide if they need help or not. If it just said 'fault by touching' that it could be a mouse or a person, they could be dying or OK. You'd never know. Seeing the person's name is rather powerful if you happen to be hunting them too

Reply:
I'm gonna be honest with you here: not a whole lot people are going to be glued to a _power monitoring_ console the whole round just in case their target gets zapped on an electrical grille:
1. you'd only see them on an HV wire, which most grilles aren't connected to
2. you'd only see it for a split second
3. there's no real guarantee they're dead
4. you wouldn't know where it happened